api/RTCPeerConnection.json

```diff
@@ -1554,6 +1554,548 @@
             "deprecated": false
           }
         },
+        "inbound_rtp_frameHeight": {
+          "__compat": {
+            "description": "<code>frameHeight</code> in 'inbound-rtp' stats",
+            "spec_url": "https://w3c.github.io/webrtc-stats/#dom-rtcinboundrtpstreamstats-frameheight",
```

Comment:
@queengooborg This is the start of what is going to replace `RTCInbountRtpStreamStats`. My current plan is to do all the values like this - so all inbounds will be `inbound_rtp_Xxxx` and there will be keys for `outbound_rtp_Xxxx` and so on that will be peers.

Do you think this is OK? Or should we have deeper grouping - e.g. a feature `inbound_rtp` and fields below it?

Reply:
This looks good to me, we can always make changes to the structure in the future if we need to!